libs/game/physics.ts

```diff
@@ -428,7 +428,9 @@ class ArcadePhysicsEngine extends PhysicsEngine {
                     );
 
                     for (const tile of collidedTiles) {
-                        s.registerObstacle(collisionDirection, tile, tm);
+                        if(!(s.flags & sprites.Flag.Destroyed)) {
```

Comment:
```suggestion
                        if(!(s.flags & SPRITE_NO_WALL_COLLISION)) {
```

Ahh I missed this in the message earlier, it should also bail out here in the same way if it became a ghost